service/src/main/java/uk/gov/hmcts/reform/fpl/utils/CustomAsyncExceptionHandler.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+import org.springframework.aop.interceptor.AsyncUncaughtExceptionHandler;
+import java.lang.reflect.Method;
+
+public class CustomAsyncExceptionHandler implements AsyncUncaughtExceptionHandler {
+
+    @Override
+    public void handleUncaughtException(
+        Throwable throwable, Method method, Object... obj) {
+
+        System.out.println("Exception message - " + throwable.getMessage());
```

Comment:
Use logger for exception logging, no `System.out.println` is allowed.

I would also expect that logging `Throwable` is enough, we do not have to log method and args.